ftNode/conf/ftNode_ftShoeLeft_yarprobotinterface.xml

```diff
@@ -0,0 +1,160 @@
+                                       0.018456480855675  -0.027680967629762   0.995373889642780  -0.378801203422740   0.405083974855969  -0.399854054343818
+                                      -0.001858452062923   0.028548921877088  -0.001296050687382   0.882390054944027  -0.021267049818161   0.012534000056372
+                                      -0.024912614354884  -0.002610851373780  -0.001454598169520  -0.013007004521544   1.198536139399159   0.056633344983102
+                                      -0.001526052208884  -0.008395096178476  -0.000888111548024   0.027328873291096  -0.022322328373427   0.963011521138837
+                                    )
+            </param>
+        </group>
+        <action phase="startup" level="5" type="attach">
+            <paramlist name="networks">
+                <elem name="ftshoe_l_ftnodedriver"> ftNodeDriver </elem>
+            </paramlist>
+        </action>
+        <action phase="shutdown" level="5" type="detach"/>
+    </device>
+
+    <!-- Analog wrappers sections, required to forward devices data on dedicated yarp ports -->
+    <!-- Analog wrappers sections - ftshoe devices wrappers -->
+    <device name="ftSensWrapper" type="analogServer">
+        <param name="period"> 10 </param>
+        <param name="name"> /ft/ftShoe_Left/analog:o </param>
+        <action phase="startup" level="5" type="attach">
+            <paramlist name="networks">
+                <elem name="FirstStrain"> ftShoe_l </elem>
+            </paramlist>
+        </action>
+        <action phase="shutdown" level="5" type="detach"/>
+    </device>
+
+    <device type="ianalogsensor_to_iwear" name="FTShoeLeftIAnalogSensorToIWear">
+        <param name="sensorName">FTShoeLeftFTSensors</param>
+        <param name="wearableName">FTShoeLeft</param>
+        <param name="numberOfChannels">6</param>
+        <param name="channelOffset">0</param>
+        <param name="wearableSensorType">ForceTorque6DSensor</param>
+        <param name="getGroundReactionFT">true</param>
+        <action phase="startup" level="5" type="attach">
+            <paramlist name="networks">
+                <elem name="FTShoeLeftIAnalogSensorToIWearLabel">ftShoe_l</elem>
+            </paramlist>
+        </action>
+        <action phase="shutdown" level="5" type="detach"/>
+    </device>
+
+    <device type="iwear_wrapper" name="FTShoeLeftIWearWrapper">
+        <param name="period">0.01</param>
+        <param name="dataPortName">/FTShoeLeft/WearableData/data:o</param>
+        <param name="rpcPortName">/FTShoeLeft/WearableData/metadataRpc:o</param>
+        <action phase="startup" level="5" type="attach">
+            <paramlist name="networks">
+                <elem name="FTShoeLeftIWearWrapperLabel">FTShoeLeftIAnalogSensorToIWear</elem>
+            </paramlist>
+        </action>
+        <action phase="shutdown" level="5" type="detach"/>
+    </device>
```

Comment:
This configuration files include device related to [`wearables`](https://github.com/robotology/wearables) repository, which is not a dependency for this repo. Indeed I would remove them from the `.xml` stored in this repo and move the full `.xml` into `wearables` repository.